tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/ProcessBasicCheck.cs

```diff
@@ -259,8 +262,15 @@ public static bool Run(ProcessInfo process, IRegistryService? registryService =
                 }
                 else
                 {
-                    Utils.WriteInfo(ContinuousProfilerNotSet);
-                    isContinuousProfilerEnabled = false;
+                    isContinuousProfilerEnabled = process.EnvironmentVariables.TryGetValue("DD_INJECTION_ENABLED", out var _);
```

Comment:
Why don't we check that it contains "profiler"?
Oh... because we don't check if it is "enabled" but if it should be loaded?

Reply:
I don't follow, this doesn't mean the profiler is enabled does it?🤔